src/Source/WMTSSource.js

```diff
@@ -87,7 +89,8 @@ class WMTSSource extends TMSSource {
             '&REQUEST=GetTile' +
             `&STYLE=${source.style || 'normal'}` +
             `&TILEMATRIXSET=${source.tileMatrixSet}` +
-            '&TILEMATRIX=%TILEMATRIX&TILEROW=%ROW&TILECOL=%COL';
+            `&TILEMATRIX=${source.tileMatrix('%TILEMATRIX')}` + 
```

Comment:
Why you use a function to set tile matrix identifier?

Reply:
According to WMTS standard, the TileMatrix parameter is not always the zoom level. For exemple, for GeoServer, it is the crs concatenate with the zoom level (ex: EPSG:4326:14)

Reply:
ok, but the callback doesn't fix it because it calls only one time, when the source is instanced.

Reply:
Actually, when i wanted to do. It was possible in TMSSource. I updated the doc, so that WMTSSource extends TMSSource and not Source.